### PR TITLE
fix: add Enum for ts

### PR DESCRIPTION
**Pull request by zhaoleipeng**
add Enum to fix typescript enum is keyword and close this issue
#64 

see the @types/js-to-java
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0b5bfba2994c91a099cd5bcfd984f6c4c39228e5/types/js-to-java/index.d.ts#L18

### CODE REVIEW

**Comment by zhaoleipeng** [comment]
@gxcsoccer, @shaoshuai0102 and @greenkeeperio-bot

File: index.js [hunk 1]
```diff
@@ -219,7 +219,7 @@ exports.abstract = function (abstractClassName, className, val) {
  *   $: { name: 'RED' }
  * }
  */
-exports.enum = function (className, name) {
+var Enum = function (className, name) {
```
Comment (gxcsoccer) [inline index.js:222]:
```js
function Enum(className, name) {
...
}
```

**Review by gxcsoccer — COMMENTED** [review]
ci 改成 10 以上？

File: index.js (hunk 1 above)
Comment (zhaoleipeng) [reply index.js:222]: ok，我调整一下。

**Comment by gxcsoccer** [comment]
+1

**Comment by gxcsoccer** [comment]
js-to-java@2.7.0

**Pull request merged**